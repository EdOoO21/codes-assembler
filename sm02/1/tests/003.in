213 114 -733 -243 875 236 -30 281 189 -866 240 -974 861 715 -40 -469 128 -521 -608 468 -37 107 713 125 -25 -187 308 763 -692 -526
