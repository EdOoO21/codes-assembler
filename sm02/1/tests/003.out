-526
-692
763
308
-187
-25
125
713
107
-37
468
-608
-521
128
-469
-40
715
861
-974
240
-866
189
281
-30
236
875
-243
-733
114
213
